contracts/addresses/addresses.go

```diff
@@ -10,5 +10,9 @@ var (
 	CeloTokenAlfajoresAddress  = common.HexToAddress("0xF194afDf50B03e69Bd7D057c1Aa9e10c9954E4C9")
 	FeeHandlerAlfajoresAddress = common.HexToAddress("0xEAaFf71AB67B5d0eF34ba62Ea06Ac3d3E2dAAA38")
 
+	CeloTokenBaklavaAddress  = common.HexToAddress("0xdDc9bE57f553fe75752D61606B94CBD7e0264eF8")
+	FeeHandlerBaklavaAddress = common.HexToAddress("0xeed0A69c51079114C280f7b936C79e24bD94013e")
```

Comment:
This address is not verified: https://celo-baklava.blockscout.com/address/0x061e5943A9491eBf996802a39E539C233118F41B?tab=contract
Neither is the registry: https://celo-baklava.blockscout.com/address/0x000000000000000000000000000000000000ce10?tab=contract

So I checked the registry entry with
```
> ETH_RPC_URL=https://baklava-forno.celo-testnet.org cast call 0x000000000000000000000000000000000000ce10 "getAddressForStringOrDie(string calldata identifier) returns (address)" FeeHandler
0xeed0A69c51079114C280f7b936C79e24bD94013e
```
✅